pkg/cmd/cli/cmd/volume.go

```diff
@@ -419,7 +507,7 @@ func (v *VolumeOptions) setVolumeMount(spec *kapi.PodSpec, info *resource.Info)
 	opts := v.AddOpts
 	containers, _ := selectContainers(spec.Containers, v.Containers)
 	if len(containers) == 0 && v.Containers != "*" {
-		fmt.Fprintf(v.Writer, "warning: %s/%s does not have any containers matching %q\n", info.Mapping.Resource, info.Name, v.Containers)
+		fmt.Fprintf(v.Out, "warning: %s/%s does not have any containers matching %q\n", info.Mapping.Resource, info.Name, v.Containers)
```

Comment:
Yes, thanks

On Aug 11, 2015, at 4:06 PM, Ravi Sankar Penta <notifications@github.com>
wrote:

In pkg/cmd/cli/cmd/volume.go
<https://github.com/openshift/origin/pull/4055#discussion_r36791987>:

> @@ -419,7 +507,7 @@ func (v *VolumeOptions) setVolumeMount(spec *kapi.PodSpec, info *resource.Info)
>  	opts := v.AddOpts
>  	containers, _ := selectContainers(spec.Containers, v.Containers)
>  	if len(containers) == 0 && v.Containers != "*" {
> -		fmt.Fprintf(v.Writer, "warning: %s/%s does not have any containers matching %q\n", info.Mapping.Resource, info.Name, v.Containers)
> +		fmt.Fprintf(v.Out, "warning: %s/%s does not have any containers matching %q\n", info.Mapping.Resource, info.Name, v.Containers)

v.Err?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4055/files#r36791987>.